lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -152,6 +152,8 @@ def resolve_association_target(params)
           }
         end
 
+        # Clear the cache so that we get the latest associations.
+        const.clear_reflections_cache
```

Comment:
This approach would mean that we essentially never have reflections cached. Do we know how expensive that could be?

Would it be worth trying to only clear the cache based on file modifications, similar to how we handle reloading on schema changes?